jest.config.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
-	testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.[jt]sx?$',
-	setupFiles: ['./src/setupTests.js'],
-	setupTestFrameworkScriptFile: './src/setupTestFramework.js',
-	testPathIgnorePatterns: ['/node_modules/', '/dist/'],
-	moduleFileExtensions: ['js', 'jsx', 'ts', 'tsx'],
-	transform: {
-		'^.+\\.(js|ts)x?$': 'babel-jest',
-	},
+    testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.[jt]sx?$',
+    setupFiles: ['./src/setupTests.js'],
+    setupTestFrameworkScriptFile: './src/setupTestFramework.js',
+    testPathIgnorePatterns: ['/node_modules/', '/dist/', '/integration/'],
```

Comment:
The only meaningful change here is on this line. Added 'integration' to ignores.